setup/helpers/abs_helper.rb

```diff
@@ -0,0 +1,284 @@
+require 'net/http'
+require 'timeout'
+require 'json'
+require 'yaml'
+
+module AbsHelper
+
+  # TODO: replace when ready
+  #ABS_BASE_URL = 'https://cinext-abs.delivery.puppetlabs.net/api/v2'
```

Comment:
I get a 504 gateway timeout when making a valid request to the abs endpoint. However, the update seems to be there because I get valid errors if parameters are missing. I have also gotten a 504 from the abs-test endpoint but most of the time it returns the expected response.